profiles/scraper.py

```diff
@@ -561,19 +616,18 @@ def parse_profile_education(self):
                     school_date_range = ""
                 else:
                     end_date = datetime(int(school_date_range.split('\n')[1].split(' ')[2]), 5, 1)
-                
+
             temp["start_date"] = start_date
             temp["end_date"] = end_date
+            print("end date", temp["end_date"])
```

Comment:
use logger instead of print if you want to log something, this prob shouldn't be logged